atom/browser/native_window_views_win.cc

```diff
@@ -90,6 +90,15 @@ bool NativeWindowViews::PreHandleMSG(
     // mode if it isn't already, always say we didn't handle the message
     // because we still want Chromium to handle returning the actual
     // accessibility object.
+    case WM_DWMCOLORIZATIONCOLORCHANGED: {
```

Comment:
I'm not sure whether `WM_DWMCOLORIZATIONCOLORCHANGED` belongs to broadcast message, most system notifications do not.

If it is we have to use a hidden window to deal with it.